source/standards/managing-sensitive-information.html.md.erb

```diff
@@ -0,0 +1,399 @@
+---
+title: How to handle sensitive code
+last_reviewed_on: 2025-10-01
+review_in: 6 months
+---
+
+# <%= current_page.data.title %>
+
+The services GDS runs has changed over time. We are increasingly engaged in more sensitive areas,
+and run many components designed to take some burden off other services.
+
+Also, working in the open benefits digital public service delivery as well as supporting an important philosophical approach to delivery.
+This results in a range of guidance, cultural norms, government policy and established practices, that encourage a measured approach to openness.
+Established examples of this are in:
+
+- Point 12 of the [service standard – make new source code open][service-standard-open-code]
+- the [GDS Way][gds-way-open-standards] and in operational practice around GDS (Including on Critical National Infrastructure components)
+- part of the [DI Source Control Management Policy][source-control] (see overview)
+
+We also have frequently published content discussing[coding in the open][2012-coding-in-open] and the benefits to GDS:
+
+- [Coding in the open][2012-coding-in-open]
+- [Why we code in the open][2017-why-we-open]
+- [Benefits of coding in the open][2017-benefits-of-open]
+- [Don’t be afraid to code in the open and how to do it securely][2017-dont-be-afraid]
+- [Barriers to openness and how to overcome them][2018-barriers-to-open]
+
+In other departments, our colleagues and peers also code in the open as an industry norm across government:
+
+- [MOJ: How making our deployment code open improved our workflow][moj-open-blog]
+- [NHS Service Standard:12. Make new source code open][nhs-service-standards]
+
+You should record your decision to code in the open using a clear description to explain which code is open and how you manage sensitive data.
```

Comment:
This is backwards. If the default position is for code to be open (which it is, based on the Service Standard etc), then you should be recording any decisions for code that is not open, not the other way around. Forcing justification for open code just makes it more likely that code will be left closed.